src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -10,11 +10,16 @@
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Remove="BasicTests\DiagnosticSettingsTests.cs" />
```

Comment:
@gucalder was this a mistake? why add these files? These files will be picked up VS project system like it did in VS2017.
you don't have to do it. But even if you did, why add two different set of entries.
Please remove these entries from the project file.